OrbitLinuxTracing/TracerThread.cpp

```diff
@@ -8,6 +8,15 @@
 #include "Utils.h"
 #include "absl/container/flat_hash_map.h"
 
+template<class T>
```

Comment:
You should put this template into an anonymous namespace,
otherwise you might run into ODR issues in case somebody
introduces the same function template somewhere else.